firebase-perf/src/main/java/com/google/firebase/perf/transport/TransportManager.java

```diff
@@ -230,6 +232,8 @@ private void finishInitialization() {
     applicationInfoBuilder = ApplicationInfo.newBuilder();
     applicationInfoBuilder
         .setGoogleAppId(firebaseApp.getOptions().getApplicationId())
+        .setProcessName(
+            ProcessDetailsProvider.getProcessDetailsProvider().getCurrentProcessDetails(appContext).getProcessName())
```

Comment:
It will look cleaner if you static import ProcessDetailsProvider.getProcessDetailsProvider then just call `getProcessDetailsProvider().get...` here

Was `getCurrentProcessDetails` internal? Is it ok to call here? Java might let you, but a lint warning complain

Reply:
Done